packages/rn-tester/android/app/src/main/java/com/facebook/react/uiapp/component/MyLegacyViewManager.kt

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.react.uiapp.component
+
+import android.graphics.Color
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.bridge.ReadableArray
+import com.facebook.react.common.MapBuilder
+import com.facebook.react.module.annotations.ReactModule
+import com.facebook.react.uimanager.SimpleViewManager
+import com.facebook.react.uimanager.ThemedReactContext
+import com.facebook.react.uimanager.ViewProps
+import com.facebook.react.uimanager.annotations.ReactProp
+
+/**
+ * Legacy View manager (non Fabric compatible) for {@link MyNativeView}
+ * components.
+ */
+@ReactModule(name = MyLegacyViewManager.REACT_CLASS)
+internal class MyLegacyViewManager(reactContext: ReactApplicationContext) :
+  SimpleViewManager<MyNativeView>() {
+  companion object {
+    const val REACT_CLASS = "RNTMyLegacyNativeView"
+    const val COMMAND_CHANGE_BACKGROUND_COLOR = 42
+  }
+
+  private val mCallerContext: ReactApplicationContext = reactContext
+
+  override fun getName(): String {
+    return REACT_CLASS
+  }
+
+  override fun createViewInstance(reactContext: ThemedReactContext): MyNativeView {
+    val view: MyNativeView = MyNativeView(reactContext)
+    view.setBackgroundColor(Color.RED)
+    return view
+  }
+
+  @ReactProp(name = ViewProps.OPACITY, defaultFloat = 1f)
+  override fun setOpacity(view: MyNativeView, opacity: Float) {
+    super.setOpacity(view, opacity)
+  }
+
+  @ReactProp(name = ViewProps.COLOR)
+  fun setColor(view: MyNativeView, color: String) {
+    view.setBackgroundColor(Color.parseColor(color))
+  }
+
+  @ReactProp(name = "cornerRadius")
+  fun setCornerRadius(view: MyNativeView, cornerRadius: Float) {
+    if (cornerRadius !== null) {
+      view.setCornerRadius(cornerRadius)
+    }
+  }
+
+  override fun getExportedViewConstants(): Map<String, Any> {
+    return mapOf("PI" to 3.14)
+  }
+
+  override fun getExportedCustomBubblingEventTypeConstants(): Map<String, Any> {
+    return MapBuilder.builder<String, Any>()
+      .put(
```

Comment:
```suggestion
    return mapOf(
```

Reply:
Hey @rshest thanks for the help!
Unfortunately, RNTesterApp does not open for neither
```
    return mapOf(
      "onColorChanged" to
        MapBuilder.of(
          "phasedRegistrationNames",
          MapBuilder.of(
            "bubbled",
            "onColorChanged",
            "captured",
            "onColorChangedCapture"
          )
        )
    )
```
nor
```
    return mapOf(
      "onColorChanged" to
        mapOf(
          "phasedRegistrationNames" to
            mapOf(
              "bubbled" to
                "onColorChanged",
              "captured" to
                "onColorChangedCapture"
            )
        )
    )
```
I applied other requested changes without any problem and the app runs correctly.